benchutils/transformers.py

```diff
@@ -57,7 +57,10 @@ def kraken2_transformer(all_rank_summary, output_rank_summaries, ranks):
 
 def metaphlan2_transformer(all_rank_summary, output_rank_summaries, ranks):
     all_ranks = pd.read_csv(all_rank_summary, sep='\t', skiprows=3)
-    def last_entry(x): return x.split('|')[-1]
+
+    def last_entry(x):
+        return x.split('|')[-1]
```

Comment:
For single-statement functions nested within another function, I prefer to use the one-line syntax in the original.